### PR TITLE
[JENKINS-33106] Dynamic CustomOrganizationFolderDescriptor registration

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-33106](https://issues.jenkins-ci.org/browse/JENKINS-33106)

@reviewbybees esp. @stephenc

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,8 +64,7 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.609.1</jenkins.version>
-    <java.level>6</java.level>
+    <jenkins.version>1.625.1</jenkins.version>
```
Comment (jglick) [inline pom.xml]: Needed for `ExtensionListListener`.

Comment (recena) [inline pom.xml]: :+1:

**Comment by recena** [comment]
@jglick LGTM but I don't have enough experience in this part.

**Comment by kohsuke** [comment]
Unrelated to the fix at hand, but what the core needs is a listener who gets notified of a new extension when it's loaded dynamically.

**Comment by kohsuke** [comment]
:bee:

**Comment by andresrc** [comment]
:bee:

**Pull request merged**